ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,210 @@
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 106</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r"/><locus to="58"/>
+                            <title type="complete" ref="LIT2361somade">Ṣoma dǝggʷā
+                          </title>
+                            <textLang mainLang="gez"/>
+                            </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                       
+
+                              
+                                <measure unit="leaf" quantity="154">2+58</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm" cert="medium">185</height>
+                                    <width unit="mm" cert="medium">140</width>
+                                    <measure unit="leaf" type="blank">2
+                                        <locus target="#ir #ii  #58v"></locus></measure>
+                                <note>The first two leaves of the small quire <ref target="q1"/> are not numbered,
+                                    and have been assigned here the number i and ii.
+                                    At the end of the volume the last folium verso is also unfoliated, assigned here the number 58v.
+                                </note>
+                                </extent>
+                                <foliation>Foliation in the upper right corner of the folia.</foliation>
+                                <collation>
+                                    <signatures>Decorated quire marks are written in the upper left corner, in the middle and the upper right corner of the first page of
```

Comment:
not 106? what is this description about?

Reply:
Am not checking further on, please make sure which manuscript you are describing